app/models/course.rb

```diff
@@ -174,15 +169,6 @@ def ratifying_provider
     accredited_provider || provider
   end
 
-  def open!
-    return if persisted? && open_on_apply
-
-    update!(
-      open_on_apply: true,
-      opened_on_apply_at: Time.zone.now,
-    )
-  end
-
   def ske_graduation_cutoff_date
     start_date - SKE_GRADUATION_CUTOFF_THRESHOLD
   end
```

Comment:
Maybe is not related to this PR but related to the open feature.

Can we use `open_for_applications?` in the `open?` method?

```ruby
  def open?
    applications_open_from.present? && applications_open_from <= Time.zone.today && exposed_in_find && application_status_open?
  end

  def open_for_applications?
    applications_open_from <= Time.zone.today
  end
```

Reply:
I'll make a note to do this in my next PR which aims to make use of `application_status` in the areas it has not been introduced.